### PR TITLE
Log frontend

### PR DESCRIPTION
**Pull request by megelismi**
Basics of logging display in place

### CODE REVIEW

File: packages/wheelhouse-frontend/src/components/development-data/DevelopmentData.css [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 .development-data-container {
   margin-top: 30px;
+  margin-left: 20px !important;
```
Comment (iameli) [inline packages/wheelhouse-frontend/src/components/development-data/DevelopmentData.css:10]: Why `!important`?

File: packages/wheelhouse-frontend/src/handlers/component-handlers/log-handlers.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  return hash;
+}; 
+
+export const _makeBrighter = (hex) => {
+  hex = hex.split(""); 
+  hex[1] = "F"; 
+  return hex.join("");
+};
+
+export const intToRGB = (i) => {
```
Comment (iameli) [inline packages/wheelhouse-frontend/src/handlers/component-handlers/log-handlers.js:16]: Love this! But let's move it to `wheelhouse-core` so we can use the same algorithm to print out the colors of the app in the CLI.

**Review by iameli — COMMENTED** [review]
Overall looks great, all this really needs is comments next to any of the `!important` lines justifying them. Or avoid them entirely, even better 😀

File: packages/wheelhouse-frontend/src/components/development-data/DevelopmentData.css (hunk 1 above)
Comment (megelismi) [reply packages/wheelhouse-frontend/src/components/development-data/DevelopmentData.css:10]: There's default margins on the grid containers from Milligram, so I have to use !important to override that styling. I can include these as comments, so it's more clear

**Pull request merged**